tests/unit/containers/edit-profile/professional-info-tab/add-professional-category-modal/AddProfessionalCategoryModal.spec.jsx

```diff
@@ -1,10 +1,11 @@
 import AddProfessionalCategoryModal from '~/containers/edit-profile/professional-info-tab/add-professional-category-modal/AddProfessionalCategoryModal'
+import userEvent from '@testing-library/user-event'
 import { renderWithProviders, selectOption } from '~tests/test-utils'
 import { act, fireEvent, screen, waitFor } from '@testing-library/react'
 import { professionalSubjectTemplate } from '~/containers/edit-profile/professional-info-tab/add-professional-category-modal/AddProfessionalCategoryModal.constants'
 import { mockAxiosClient } from '~tests/test-utils'
 import { URLs } from '~/constants/request'
-import { vi } from 'vitest'
+import { describe, expect, vi } from 'vitest'
```

Comment:
```suggestion
import { vi } from 'vitest'
```
You can do this and it will work

Reply:
ok